source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -268,7 +269,9 @@ HttpConnectionManagerConfig::HttpConnectionManagerConfig(
       stats_(Http::ConnectionManagerImpl::generateStats(stats_prefix_, context_.scope())),
       tracing_stats_(
           Http::ConnectionManagerImpl::generateTracingStats(stats_prefix_, context_.scope())),
-      use_remote_address_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(config, use_remote_address, false)),
+      use_remote_address_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(
```

Comment:
I'm wondering if we should have a new macro called `PROTOBUF_GET_WRAPPED_OR_PROFILE_DEFAULT` that will take the 3rd parameter and fetch it from `DefaultsProfileSingleton::get()`. I don't feel strong about it, just something to think about.